src/Components/PresentationalComponents/InsightsSystemRule/InsightsSystemRule.js

```diff
@@ -63,3 +63,24 @@ export const InsightsNoSystemRule = ({ cve }) => {
 InsightsNoSystemRule.propTypes = {
     cve: PropType.string
 };
+
+export const InsightsNotVulnerable = ({ cve, mitigationReason }) => {
+    return (
+        <TextContent>
+            <Text component={TextVariants.h3}>
+                <FormattedMessage {...messages.notVulnerableBoxTitle} />
+            </Text>
+            <Text component={TextVariants.p} className="pf-u-mb-sm">
```

Comment:
Why this class name is needed?

Reply:
To make the paragraphs separated by 8px as in mockups.